src/components/project/home-block.vue

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="project-home-block">
+    <div class="title">
+      <router-link :to="projectPath(project.id)">{{ project.name }}</router-link>
+      <span v-if="project.keyId" class="encrypted badge"
+        :title="$t('encryptionTip')">
+        <span class="icon-lock project-icon"></span>
+        {{ $t('encrypted') }}
+        </span>
+    </div>
+    <table v-if="visibleForms != null" class="project-form-table table">
+      <project-form-row v-for="form of visibleForms" :key="form.xmlFormId" :form="form"/>
+    </table>
+    <div v-if="showExpander">
+      <span class="expand-button" @click.prevent="toggleExpanded">
```

Comment:
I think that using a `<span>` as a button here might not work well from an accessibility standpoint. It might help to specify `role="button"` (the [related page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) on MDN has useful guidance in general), but even then, I don't know that a keyboard user would be able to navigate to the element. One strategy that I've used (and that comes up sometimes in Bootstrap) is to render the element as a link (specify `href="#" role="button"` or maybe even just `role="button"`). You could then use the `text-link` mixin. (I'm sure there are also other approaches.)